### PR TITLE
multiple setup script improvment

### PR DESCRIPTION
**Pull request by yodapotatofly**
- memory option for logstash and elastic
Using the mem_limit on the container, because using ES_JAVA_OPTS in not recommended in production

-  option to change restart behavior
this can be used to prevent containers from restarting on boot #357 

- bility to load docker images from local tarballs

### CODE REVIEW

File: docker/easy-setup.sh [hunk 1]
```diff
@@ -62,12 +62,31 @@ function Help(){
     echo -e "       Defines the path where Elasticsearch will store it's data. The path must already exists and the current user must have write permissions. Default will be in a named docker volume ('/var/lib/docker')"
     echo -e "       The interactive prompt regarding this option will be skipped\n"
     echo -e " --es-memory"
-    echo -e "       Amount of memory to give to the elasticsearch java heap. Default is '2G'. Accepted units are 'm','M','g','G'. ex \"--es-memory 512m\" or \"--es-memory 4G\". Default is '2G'\n"
+    echo -e "       Amount of memory to give to the elasticsearch java heap. Accepted units are 'm','M','g','G'. ex \"--es-memory 512m\" or \"--es-memory 4G\". Default is '2G'\n"
+    echo -e " --ls-memory"
+    echo -e "       Amount of memory to give to the logstash java heap. Accepted units are 'm','M','g','G'. ex \"--es-memory 512m\" or \"--es-memory 4G\". Default is '2G'\n"
+    echo -e " --restart-mode"
+    echo -e "       'no', 'always', 'on-failure','unless-stopped'"
```
Comment (pevma) [inline docker/easy-setup.sh:69]: Can you add description of the options please?

File: docker/easy-setup.sh [hunk 2]
```diff
@@ -587,6 +617,29 @@ esac
 
 echo
 
+################
+# RESTART MODE #
+################
+
+echo -e "Do you want the containers to restart automatically on startup? [Y/n] "
```
Comment (pevma) [inline docker/easy-setup.sh:624]: Is this taking into account  non interractive setup ?

Comment (yodapotatofly) [reply docker/easy-setup.sh:624]: yes, with more precision through the "--restart-mode"

File: docker/easy-setup.sh (hunk 1 above)
Comment (yodapotatofly) [reply docker/easy-setup.sh:69]: done.

**Pull request merged**